iota-client/tests/api_test.rs

```diff
@@ -378,7 +378,7 @@ async fn test_prepare_transfers_no_value() {
         transfers.push(Transfer {
             address: Address::zeros(),
             value: 0,
-            message: "".to_string(),
+            message: None,
```

Comment:
the tag field is needed in transfer implementation.

Reply:
Also can we set the timestamp here or not needed?

Reply:
No, timestamp should be handled by method itself. Regarding tag with above comment, I'll push it in a moment then.